german/the-black-hack.md

```diff
@@ -26,38 +26,39 @@ The Black Hack (TBH) ist ein traditionelles Tischrollenspiel, das mit Papier, St
 
 Wann immer Charaktere versuchen, etwas zu tun, das möglicherweise scheitern könnte, wird der Erfolg mit einer Probe auf einen Eigenschaftswert getestet. Um eine Probe erfolgreich zu bestehen, müssen Spieler mit einem W20 ***unter*** den entsprechenden Eigenschaftswert würfeln.
 
-Monster würfeln keine Proben - Spielercharaktere müssen ihren Angriffen ausweichen, indem sie eine Probe bestehen. Das einzige, was Monster würfeln, ist ihr Schaden.
+Monster würfeln keine Proben - Spielercharaktere müssen ihren Angriffen ausweichen, indem sie eine Probe bestehen. Das einzige, das Monster würfeln, ist ihr Schaden.
 
 ## Eigenschaftswerte auswürfeln
 
-**Eigenschaftswerte** werden mit 3W6 in der folgenden Reihenfolge bestimmt: **Stärke (STÄ), Geschick (GES), Ausdauer (AUS), Intelligenz (INT), Weisheit (WEI), Charisma (CHA)**. Wenn ein gewürfelter Eigenschaftswert 15+ beträgt, muss der folgende mit 2W6 + 2 bestimmt werden, danach wird wieder mit 3W6 gewürfelt, bis eine weitere 15+ fällt. Wenn die Werte für alle Eigenschaften bestimmt wurden, dürfen zwei davon miteinander getauscht werden.
+**Eigenschaftswerte** werden mit 3W6 in der folgenden Reihenfolge bestimmt: **Stärke (STÄ), Geschick (GES), Konstitution (AUS), Intelligenz (INT), Weisheit (WEI), Charisma (CHA)**. Wenn ein gewürfelter Eigenschaftswert 15+ beträgt, muss der folgende mit 2W6 + 2 bestimmt werden, danach wird wieder mit 3W6 gewürfelt, bis eine weitere 15+ fällt. Wenn die Werte für alle Eigenschaften bestimmt wurden, dürfen zwei davon miteinander getauscht werden.
```

Comment:
Ja, diese Argumentation halte ich für stichhaltig. Ich konnte es nur nicht so in Worte fassen, hatte ja aber schon ganz am Anfang meine Tendenz zu Konstitution geäußert.

Weiteres Argument pro Konstitution: Ausdauer ist keine Übersetzung von constitution sondern von stamina!